src/utils/url.js

```diff
@@ -157,3 +157,12 @@ export const autocompleteUrl = (
         value.indexOf('.') === -1 ? append : !value.match(/.+\..+\.+./g) ? shortAppend : ''
       }`
     : '';
+
+export const encodeImageUri = (uri: string): string => {
```

Comment:
Adding a test for this function will prove that it works correctly.

Reply:
Added the tests now. Thank you.